lib/pages/race/race_page.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:flutter_map_location_marker/flutter_map_location_marker.dart';
+import 'package:regatta_buddy/modals/action_button.dart';
+import 'package:regatta_buddy/modals/actions_dialog.dart' as actions_dialog;
+import 'package:regatta_buddy/pages/race/race_statistics.dart';
+import 'package:regatta_buddy/utils/constants.dart' as constants;
+import 'package:regatta_buddy/utils/externalAPIConstants.dart' as api_constants;
+import 'package:regatta_buddy/widgets/app_header.dart';
+import 'package:regatta_buddy/widgets/rb_notification.dart';
+import 'package:uuid/uuid.dart';
+
+class RacePage extends StatefulWidget {
+  const RacePage({super.key});
+
+  static const String route = '/race';
+
+  @override
+  State<RacePage> createState() => _RacePageState();
+}
+
+class _RacePageState extends State<RacePage> {
+  final int _notificationTimeInSeconds = 10;
+  List<RBNotification> activeNotifications = [];
+  List<ActionButton> raceActions = [];
+
+  void addNotification(String title) {
```

Comment:
are these notifications in app or in phone system?

Reply:
It is just within the app but I saw really simple lib for phone notifications. We might use it in the future